src/Compilers/Helpers/GlobalAssemblyCacheHelpers/GacFileResolver.cs

```diff
@@ -11,7 +11,7 @@
 namespace Microsoft.CodeAnalysis.Scripting
 {
     /// <summary>
-    /// Extends <see cref="MetadataFileReferenceResolver"/> to enable resolution of assembly
+    /// Extends MetadataFileReferenceResolver to enable resolution of assembly
```

Comment:
Yeah, you should complain to the compiler lead to get that fixed, where in the case of an ambiguity like this prefer the one in the assembly vs. one external to it.